### PR TITLE
Miscellaneous, boring cleanups

### PR DESCRIPTION
**Pull request by rossabaker**
These are uninteresting cleanups that came out of experimentation on #390.  Separated out, as they're all anticipated to be uncontroversial and positive.

### CODE REVIEW

File: argonaut/src/main/scala/org/http4s/argonaut/Parser.scala [hunk 1]
```diff
@@ -1,51 +0,0 @@
-/*
```
Comment (bryce-anderson) [inline argonaut/src/main/scala/org/http4s/argonaut/Parser.scala]: The deletion of this is a pleasant surprise.

File: blaze-core/src/main/scala/org/http4s/blaze/util/BodylessWriter.scala [hunk 2]
```diff
@@ -1,4 +1,6 @@
-package org.http4s.blaze.util
```
Comment (bryce-anderson) [inline blaze-core/src/main/scala/org/http4s/blaze/util/BodylessWriter.scala]: Any particular reason for this change? I don't think I had a consistent style, but I don't see a reason associated with this one.

File: core/src/main/scala/org/http4s/package.scala [hunk 3]
```diff
@@ -75,9 +75,10 @@ package object http4s {
       * services will have the opportunity to handle the request.
       * See [[Fallthrough]] for more details.
       */
-    val notFound: Task[Response] = Task.now(Response(Status.NotFound)
-                                               .withAttribute(Fallthrough.fallthroughKey, ())
-                                               .withBody("404 Not Found.").run)
+    val notFound: Task[Response] =
+      Response(Status.NotFound)
+        .withAttribute(Fallthrough.fallthroughKey, ())
+        .withBody("404 Not Found.")
```
Comment (bryce-anderson) [inline core/src/main/scala/org/http4s/package.scala]: It's not as fun clean, but I kind of liked memorizing the result of this computation considering its constant and all.

File: blaze-core/src/main/scala/org/http4s/blaze/util/BodylessWriter.scala (hunk 2 above)
Comment (bryce-anderson) [inline blaze-core/src/main/scala/org/http4s/blaze/util/BodylessWriter.scala]: NM. I jumped the gun and noticed you've changed quite a few of them.

**Comment by bryce-anderson** [comment]
I like it. 👍

File: core/src/main/scala/org/http4s/package.scala (hunk 3 above)
Comment (rossabaker) [inline core/src/main/scala/org/http4s/package.scala]: That's a fair point.  I'll revert this one, but with a comment.

**Comment by rossabaker** [comment]
I guess maybe we do still need that argonaut parser.

**Comment by rossabaker** [comment]
Now I remember.  jawn-argonaut is compiled against 6.1 for Scalaz 7.1.  I added jawn support to argonaut in 6.2.  Later, argonaut-6.1a appeared, and there is no jawn support for that.  We're still stuck with that one.

**Pull request merged**